### PR TITLE
In the examples, listen on 0.0.0.0 instead of 127.0.0.1 #664

### PR DESCRIPTION
**Pull request by schitcrafter**
#664

### CODE REVIEW

File: examples/poem/mongodb/src/main.rs [hunk 1]
```diff
@@ -60,13 +60,13 @@ async fn main() -> io::Result<()> {
     }
     tracing_subscriber::fmt::init();
 
-    let mongodb = Client::with_uri_str("mongodb://127.0.0.1:27017")
+    let mongodb = Client::with_uri_str("mongodb://0.0.0.0:27017")
```
Comment (viell-dev) [inline examples/poem/mongodb/src/main.rs:63]: Connecting to 0.0.0.0 isn't strictly speaking valid. (But, in practice, it should be equivalent to connecting to localhost.)

File: examples/poem/redis-session/src/main.rs [hunk 2]
```diff
@@ -20,13 +20,13 @@ async fn main() -> Result<(), std::io::Error> {
     }
     tracing_subscriber::fmt::init();
 
-    let client = Client::open("redis://127.0.0.1/").unwrap();
+    let client = Client::open("redis://0.0.0.0/").unwrap();
```
Comment (viell-dev) [inline examples/poem/redis-session/src/main.rs:23]: Connecting to 0.0.0.0 isn't strictly speaking valid. (But, in practice, it should be equivalent to connecting to localhost.)

File: examples/poem/websocket-chat/src/main.rs [hunk 3]
```diff
@@ -42,7 +42,7 @@ fn index() -> Html<&'static str> {
             sendForm.hidden = false;
             msgsArea.hidden = false;
             msgInput.focus();
-            ws = new WebSocket("ws://127.0.0.1:3000/ws/" + nameInput.value);
+            ws = new WebSocket("ws://0.0.0.0:3000/ws/" + nameInput.value);
```
Comment (viell-dev) [inline examples/poem/websocket-chat/src/main.rs:45]: Connecting to 0.0.0.0 isn't strictly speaking valid. (But, in practice, it should be equivalent to connecting to localhost.)

**Review by viell-dev — CHANGES_REQUESTED** [review]
Looks like you did a search-and-replace. You also replaced some outgoing IP addresses.

**Comment by sunli829** [comment]
thanks 😂

**Pull request merged**